### PR TITLE
Avoid using eval

### PR DESCRIPTION
**Pull request by tkoyama010**
Hi. This is the change of avoid using eval. I changed eval to safer ast.literal_eval.
https://docs.python.org/3.8/library/ast.html#ast.literal_eval
Thank you for reading.

### CODE REVIEW

File: examples/mayavi/interactive/lorenz_ui.py [hunk 1]
```diff
@@ -108,7 +109,7 @@ def get_vel(self, comp):
             g = scipy.__dict__
             x, y, z = self.points
             s, r, b = self.s, self.r, self.b
-            val = eval(func_str, g,
+            val = ast.literal_eval(func_str, g,
```
Comment (prabhuramachandran) [inline examples/mayavi/interactive/lorenz_ui.py]: This is incorrect and will not work, in cases like this `eval` is really better than `ast.literal_eval`.

Comment (tkoyama010) [reply examples/mayavi/interactive/lorenz_ui.py]: Thank you for your comment 👍

Comment (tkoyama010) [inline examples/mayavi/interactive/lorenz_ui.py]:
```suggestion
            val = eval(func_str, g,
```

**Comment by tkoyama010** [comment]
Hmm... it seems CI problem.
https://travis-ci.org/enthought/mayavi/jobs/629317038#L52

**Pull request merged**